src/python/ensembl/io/genomio/gff3/simplifier.py

```diff
@@ -441,3 +445,50 @@ def remove_cds_from_pseudogene(self, gene: SeqFeature) -> None:
             transcript.sub_features = new_subfeats
             gene_subfeats.append(transcript)
         gene.sub_features = gene_subfeats
+
+
+    def normalize_mirna(self, gene: SeqFeature) -> List[SeqFeature]:
+        """Returns gene representations from a miRNA gene that can be loaded in an Ensembl database.
+
+        Change the representation from the form `gene[ primary_transcript[ exon, miRNA[ exon ] ] ]`
+        to `gene[ primary_transcript[ exon ] ]` and `gene[ miRNA[ exon ] ]`
+        """
+
+        transcript = gene.sub_features
+        if len(transcript) == 0 or transcript[0].type != "primary_transcript":
+            return []
+        if len(transcript) > 1:
+            raise GFFParserError(f"Gene has too many sub_features for miRNA {gene.id}")
+        
+        logging.debug(f"Formatting miRNA gene {gene.id}")
+        
+        primary = transcript[0]
+        new_genes = []
+        new_primary_subfeatures = []
+        num = 1
+        for sub in primary.sub_features:
+            if sub.type == "exon":
+                new_primary_subfeatures.append(sub)
+            elif sub.type == "miRNA":
+                new_gene_id = f"{gene.id}_{num}"
+                num += 1
+                new_gene = SeqFeature(sub.location, "gene", id=new_gene_id)
+                new_gene.qualifiers = {"source": sub.qualifiers["source"], "ID": new_gene_id}
+                new_gene.sub_features = [sub]
+                new_genes.append(new_gene)
+            else:
+                raise GFFParserError(f"Unknown subtypes for miRNA features: {sub.id}")
+        primary.sub_features = new_primary_subfeatures
+
+        if not new_genes:
+            raise GFFParserError(f"Could not parse a primary_transcript for {gene.id}")
+
+        all_genes = [gene] + new_genes
+
+        # Normalize like normal genes
```

Comment:
```suggestion
        # Normalize like other genes
```
Just to not classify miRNA as abnormal :sweat_smile: